srcts/src/main.ts

```diff
@@ -1480,13 +1480,14 @@ function main(): void {
 
     // If the given tabset has no active tabs, select the first one
     function ensureTabsetHasVisibleTab($tabset) {
-      if ($tabset.find("li.active").not(".dropdown").length === 0) {
+      const inputBinding = $tabset.data("shiny-input-binding");
+
+      if (!inputBinding.getValue($tabset)) {
```

Comment:
```suggestion

      // Use the getValue() method to avoid duplicating the CSS selector
      // for querying the DOM for the currently active tab
      if (!inputBinding.getValue($tabset)) {
```